1-mandatory/3-reading-list.js

```diff
@@ -47,7 +77,12 @@ As an example for this exercise, you might do the following steps
 - Modify the books so that they have the correct alreadyRead value
 - All tests should turn green!!
 **/
-
+function logBooksStatus() {
+  books.forEach((book) => {
+  const bookStatus = book.alreadyRead ? "You've already read" : "You still need to read";
+  console.log(`${bookStatus} "${book.title}" by ${book.author}.`);
+  });
+  }
 /* ======= TESTS - DO MODIFY (!!!) =====
```

Comment:
I think here you need to modify the test and write a new test to let your test pass => green, you should read the instructions carefully